hugo/layouts/partials/home/our-links.html

```diff
@@ -1,10 +1,6 @@
-<p>
-    {{ $home := $.Site.Data.home.home }}
-    {{ range where $home.hero.links "url" "ne" "#"  }}
```

Comment:
If we knew for sure it's gonna be needed I'd add it, yes. But I'm not into putting things "just in case we need them in the future". If they want to hide one of the buttons, it will require action from our side, so it's not much more overhead for us.